<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>824</width>
    <height>595</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QSkyDrive Explorer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QListView" name="listView"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionBack"/>
   <addaction name="actionForward"/>
   <addaction name="actionHome"/>
   <addaction name="separator"/>
   <addaction name="actionNewFolder"/>
   <addaction name="separator"/>
   <addaction name="actionUpload"/>
   <addaction name="actionRemove"/>
  </widget>
  <action name="actionBack">
   <property name="icon">
    <iconset theme="back">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </action>
  <action name="actionForward">
   <property name="icon">
    <iconset theme="forward">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>Forward</string>
   </property>
  </action>
  <action name="actionHome">
   <property name="icon">
    <iconset theme="go-home">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>Home</string>
   </property>
  </action>
  <action name="actionNewFolder">
   <property name="icon">
    <iconset theme="folder-new">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>New Folder</string>
   </property>
  </action>
  <action name="actionUpload">
   <property name="icon">
    <iconset theme="add">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>Upload</string>
   </property>
   <property name="toolTip">
    <string>Upload</string>
   </property>
  </action>
  <action name="actionRemove">
   <property name="icon">
    <iconset theme="remove"/>
   </property>
   <property name="text">
    <string>Remove</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
